boards.txt

```diff
@@ -7705,6 +7705,120 @@ lolin_c3_mini.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 
+lolin_c3_pico.name=LOLIN C3 Mini
```

Comment:
LOLIN C3 Mini is already taken, by lolin_c3_mini board, I think it should be "Pico" instead of Mini
```suggestion
lolin_c3_pico.name=LOLIN C3 Pico
```